### ISSUE TITLE
First run in Bionic. Kupfer failed to start. [1.absence of gobject-introspection 2.print url syntax error "()"]. 

### ISSUE THREAD

**Issue by JuPlutonic** [kupferlauncher/kupfer#110]
### Environment

Kupfer Version: 319
Window Manager: Openbox
Desktop Environment: LXDE
Linux Distribution: Ubuntu Bionic x64
Kupfer installed from Synaptic (official Canonical repos)

### Actual Behaviour KUPFER failed to start, got bug:

```
Traceback (most recent call last):
  File "/usr/share/kupfer/kupfer.py", line 22, in <module>
    main.main()
  File "/usr/share/kupfer/kupfer/main.py", line 177, in main
    gtkmain(browser_start, quiet)
  File "/usr/share/kupfer/kupfer/main.py", line 142, in gtkmain
    import gi
  File "/opt/.linuxbrew/Cellar/python/3.6.5/lib/python3.6/site-packages/gi/__init__.py", line 39
    print url
            ^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print(url)?

```

### Additional information

python3 -V                   16:37  13.05.18
Python 3.6.5
which python3                16:39  13.05.18 
/opt/.linuxbrew/bin/python3
intltool 0.51.0 installed from Linuxbrew 

Before reported above bug I got another bug what declined KUPFER to start 
```ModuleNotFoundError: No module named 'gi'```
and I had to install 
pip3 install gi          16:33  13.05.18

**Comment by JuPlutonic** [comment]
On distros python3

**Comment by KarolBedkowski** [comment]
"gi" from pypi is unrelated (gi 1.2 - Command line to private gist. Example: gi.py myFile).
You need "Python 3 bindings for gobject-introspection libraries" - on Debian python3-gi package.
Probably installing "pip install PyGObject" solve this problem. Of course you have to remove "gi" library.

**Comment by JuPlutonic** [comment]
Ok, pip3 install PyGObject for linuxbrew / sudo pip3 install PyGObject for ubuntu did the job.
Please, close the issue.
Now my setup getting errors with GTK3 when  I'm trying to run `kupfer` and atk when I'm requiring
```
from gi import Repository 
r.require('Gtk')
```
and when trying to install `brew install gtk+3`.
PS If someone gets `ValueError: Namespace Gtk not available for version 3.0` comment here maybe I'll find a clue.

**Comment by KarolBedkowski** [comment]
In ubuntu there is also package python3-gi - should be used instead of pypi version.
For GTK support should be installed gir1.2-gtk-3.0
